### PR TITLE
update missed requirement [skip ci]

### PR DESCRIPTION
**Pull request by chesterxgchen**

### CODE REVIEW

File: examples/psi/requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-openmined.psi==1.0.3
\ No newline at end of file
+openmined.psi==1.l.0
```
Comment (YuanTingHsieh) [inline examples/psi/requirements.txt:1]: is this a typo?

should it be `1.1.0`?

**Review by YuanTingHsieh — APPROVED** [review]
thanks, LGTM

**Review by IsaacYangSLA — APPROVED** [review]
LGTM.

**Pull request merged**